index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700;900&family=Playfair+Display&family=Poppins&display=swap"
+        rel="stylesheet">
+
+    <link rel="stylesheet" href="styles.css">
+    <title>home-conference</title>
+</head>
+
+<body>
+    <header>
+        <nav class="desk_menu">
+            <ul class="social-icons">
+                <li><a class="top-a" href="https://www.twitter.com/"><img src="./images/ic_twitter.svg"></a></li>
+                <li><a class="top-a" href="https://www.instagram.com/"><img src="./images/ic_linkedin.svg"></a></li>
+                <li><a class="top-a" href="#join">English</a></li>
+                <li><a class="top-a" href="#sponsor">My page</a></li>
+                <li><a class="top-a" href="#news">Logout</a></li>
+            </ul>
+
+        </nav>
+
+        <nav class="nav_menu">
+
+            <img class="logo" src="./images/logo.PNG" alt="logo">
+            <ul class="desk-bar">
+                <li><a class="top-b" href="./">Home</a></li>
+                <li><a class="top-b" href="./about.html">About</a></li>
+                <li><a class="top-b" href="#join">Join</a></li>
+                <li><a class="top-b" href="#sponsor">Sponsor</a></li>
+                <li><a class="top-b" href="#news">News</a></li>
+            </ul>
+        </nav>
+
+    </header>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please create the `button` highlighted in the image below as shown in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏 

![image](https://user-images.githubusercontent.com/63915024/203847664-9c329d16-685a-4d1d-bee4-b678abc47a19.png)